WordPress/gutenberg
# Show shadow tool by default under global styles

Fixes #61952

## What?
This PR shows the shadow tool by default under global styles.

![image](https://github.com/WordPress/gutenberg/assets/54422211/f5a6437f-e1d8-4a12-bf40-181c39c6e39d)

## Why?

I think available tools should be displayed by default in the global styles, like the Typography panel and color panel.

## How?
Changed the value of the `shadow` property from false to true in the `DEFAULT_CONTROLS` variable. This should not affect the block sidebar shadow panel.

## Testing Instructions

- Go to the Site Editor > Global Styles > Blocks > Button
- The shadow panel should be visible by default.
- Open the block sidebar for the Button block. Shadow panels should not be visible by default.
- (Additional testing) In the Button block, make the following changes to display the shadow panel by default:
  ```diff
  diff --git a/packages/block-library/src/button/block.json b/packages/block-library/src/button/block.json
  index 740f3e50f8..8d378344c3 100644
  --- a/packages/block-library/src/button/block.json
  +++ b/packages/block-library/src/button/block.json
  @@ -99,7 +99,10 @@
                  },
                  "reusable": false,
                  "shadow": {
  -                       "__experimentalSkipSerialization": true
  +                       "__experimentalSkipSerialization": true,
  +                       "__experimentalDefaultControls": {
  +                               "shadow": true
  +                       }
                  },
                  "spacing": {
                          "__experimentalSkipSerialization": true,
  ```

- In the Button block's sidebar, the shadow panel should be visible by default.
